language/diem-framework/modules/0L/Teams.move

```diff
@@ -0,0 +1,302 @@
+///////////////////////////////////////////////////////////////////////////
+// 0L Module
+// Teams 
+///////////////////////////////////////////////////////////////////////////
+// Used for coordinating "Teams", equivalent to delegation
+// Teams are groups which engage in work.
+// some of that work is automated such as validation.
+// A Team has a collective "consensus weight", which is used for voting in consensus
```

Comment:
know it might be late but just a thought: should we use consensus weight only for being in top 100 and consider all teams to be equal consensus weight. there is a slight probability of malicious node having higher consensus weight

Reply:
This is interesting. Something for @zmanian

Reply:
@sm86 Can you ticket this as an idea to research in Issues, for Phase 2?